lib/com/vaadin/components/gwt/polymer/client/Polymer.java

```diff
@@ -17,7 +16,7 @@
      * the appropriate <import> tag in the document header
      */
     public static void ensureTag(String tagName) {
-        ensureHTMLImport(ImportsMap.getInstance().get(tagName));
+        ensureHTMLImport(tagName + "/" + tagName + ".html");
```

Comment:
This is not true in all cases, see CascadedAnimationElement.java
We should have a way to figure out the CascadedAnimationElement.SRC from CascadedAnimationElement.TAG